heroku/cli
# migrate(W-14180047): pg-v5: Upgrade pg:diagnose

This PR moves `pg:diagnose` to the cli and migrates to TypeScript

### Test Instructions
1. checkout this branch and run `yarn && yarn build && cd packages/cli`
2. run `./bin/run pg:diagnose -a YOUR_APP YOUR_DATABASE` 
3. Note the UUID at the top of the report and verify the report looks like the report in the original CLI
4. display an existing report by running `./bin/run pg:diagnose -a YOUR_APP THE_UUID`
5. Verify the previous report is displayed.
